docs/tutorial_easybuild.rst

```diff
@@ -0,0 +1,81 @@
+============================================
+Tutorial 5: Using the EasyBuild build system
+============================================
+
+In this tutorial we will present how to build and install software in a ReFrame test given an EasyBuild easyconfig file.
+The example uses the configuration file presented in :doc:`tutorial_basics`, which you can find in ``tutorials/config/settings.py``.
+We also assume that the reader is already familiar with the concepts presented in the basic tutorial and has a working knowledge of EasyBuild.
+Finally, to avoid specifying the tutorial configuration file each time you run the test, make sure to export it here:
+
+.. code:: bash
+
+   export RFM_CONFIG_FILE=$(pwd)/tutorials/config/mysettings.py
+
+
+
+Basic test using EasyBuild
+----------------------------
+
+Let's consider a simple ReFrame test that installs ``bzip2-1.0.6`` given the easyconfig `bzip2-1.0.6.eb <https://github.com/eth-cscs/production/blob/master/easybuild/easyconfigs/b/bzip2/bzip2-1.0.6.eb>`__ and checks that the installed version is correct.
+The following code block shows the check, highlighting the lines specific to this tutorial:
+
+
+.. literalinclude:: ../tutorials/build_systems/easybuild/eb_test.py
+   :emphasize-lines: 15-17,21
+
+The test looks pretty standard except for the highlighted blocks.
+Let's have a look first to the block in the ``BZip2Check`` class.
+
+The first thing is to specify that the EasyBuild build system will be used.
+This is done by setting :attr:`~reframe.core.pipeline.RegressionTest.build_system` to ``'EasyBuild'``.
+Then, the software to be installed is passed as a list to :attr:`~reframe.core.buildsystems.EasyBuild.easyconfigs`.
+Here only one easyconfig is given, but more than one can be passed.
+Finally, through :attr:`~reframe.core.buildsystems.EasyBuild.options`, command line options can be passed to the ``eb`` executable.
+In this test we pass ``-f`` to make sure that ``bzip2`` will be built even if the module already exists externally.
+
+For this test, ReFrame generates the following command to build and install the easyconfig:
+
+.. code-block:: console
+
+   eb bzip2-1.0.6.eb -f
```

Comment:
I think that we are missing the environment variables that are set, aren't we? Because in the next line we say that everything will be installed in the `{stagedir}/easybuild`, so the readers will wonder how this is happening?

Reply:
We need also to mention that ReFrame expects EasyBuild to be installed already on the system.